dynamodb/src/test/scala/akka/stream/alpakka/dynamodb/ItemSpec.scala

```diff
@@ -49,15 +50,31 @@ class ItemSpec extends TestKit(ActorSystem("ItemSpec")) with AsyncWordSpecLike w
         .map(_.getItem.get("data").getS shouldEqual "test4data")
     }
 
-    "5) put an item and read it back in a batch" in {
+    "5) put two items in a batch" in {
```

Comment:
This is not what this tests does, right?

Reply:
Why not? That's exactly what it does:
```scala
val batchWriteItemRequest = new BatchWriteItemRequest().withRequestItems(
  Map(
    tableName ->
    List(
      new WriteRequest(new PutRequest().withItem((keyMap("B", 0) + ("data" -> S(test5Data))).asJava)),
      new WriteRequest(new PutRequest().withItem((keyMap("B", 1) + ("data" -> S(test5Data))).asJava))
    ).asJava
  ).asJava
)
```